### ISSUE TITLE
Zabbix can't recieve data when using agent encryption

### ISSUE THREAD

**Issue by j91321** [zarplata/zabbix-agent-extension-elasticsearch#19]
I'm using zabbix-agent-extention-elasticsearch 20180831.32_7cbc5d1  and I noticed that Zabbix will not receive most of the items when only encrypted communication is allowed from host.

As a workaround allow No encryption from host together with PSK or Certificate.

Not sure if this is a problem in the newer versions too, feel free to close this issue if it is already fixed. Just wanted to save somebody time debugging the issue, because it's not really mentioned anywhere.

**Comment by tears-of-noobs** [comment]
@j91321 Hi. Do you talk about communication between the extension and elasticsearch node through HTTPS?

**Comment by tears-of-noobs** [comment]
@j91321 Seems I've understood what you mean. You talk about - https://www.zabbix.com/documentation/4.2/manual/encryption. It's a pity but now we don't support any of the encryption features. The `zabbix-sender` functionality of the extension implemented in https://github.com/adubkov/go-zabbix and there is only one solution  - to make a PR with encryption support to the library.

**Comment by j91321** [comment]
@tears-of-noobs Yes that's what I mean. If I'll have time I'll look into it, but I doubt my go skills are good enough. Anyway think it would be worth adding this to the readme, in case anybody else encounters this problem. 

For me the workaround of allowing no encryption from host for extension and PSK for agent is good enough and acceptable solution. This way only the data collected by the extension goes unencrypted.
